htdocs/compta/cashcontrol/report.php

```diff
@@ -135,6 +135,8 @@
 	dol_print_error('Value for key = '.$key.' not supported');
 	exit;
 }*/
+if ($object->date_valid==null) $object->date_valid=dol_now();
+$sql .= " AND f.date_closing BETWEEN '".$db->idate($object->date_creation)."' AND '".$db->idate($object->date_valid)."'";
```

Comment:
Don't you think we should use a  date of payment instead of f.date_closing ?
If on an invoice, a payment is done the morning, next one the second day, invoice should appears twice and first payment amount on 1 day and second payment amount on second day.